platforms/allwinner-d1/core/src/drivers/sharp_display.rs

```diff
@@ -132,12 +123,14 @@ impl SharpDisplay {
 /// One entire frame, stored one bit per pixel
 struct FullFrame {
     frame: [[u8; WIDTH_BYTES]; HEIGHT],
+    dirty_line: [bool; HEIGHT],
```

Comment:
tiny nit: this naming is a little unclear to me, i would have maybe called it `dirty_lines` to make it clear that it's a map of _all_ lines that might be dirty. looking at the code below where it's referenced i mistakenly thought we were iterating over the pixels of a single dirty line, at first.

Reply:
Side Thing, i wonder if we could golf this into using a bitmap instead of a bytemap. probably not actually important as we have a whopping Entire Gigabyte of RAM...

Reply:
Yeah, we definitely can! I make some notes about this in the code, it'd probably be a little more computationally intense to do all the bithax, but I ran out of brain juice to do it in this PR.

The hope is that `FrameChunk` can have more variants for different color depth in the future (and ideally we can convert across them), so we could have an 8bpp mono mode when you care more about perf, and a 1bpp mono mode when you care about size (and also 8/16/24/32bpp color modes).